src/Settings/Settings/SettingsTest.php

```diff
@@ -0,0 +1,293 @@
+				'inputFieldLabel' => \__('input required label', 'eightshift-forms'),
+				'inputFieldHelp' => __('help field input required', 'eightshift-forms'),
+				'inputType' => 'text',
+				'inputValue' => $this->getOptionValue('input-required-id'),
+				'inputIsRequired' => true,
+			],
+			[
+				'component' => 'textarea',
+				'textareaId' => $this->getSettingsName('textarea-id'),
+				'textareaFieldLabel' => \__('textarea label', 'eightshift-forms'),
+				'textareaFieldHelp' => __('help field textarea', 'eightshift-forms'),
+				'textareaValue' => $this->getOptionValue('textarea-id'),
+			],
+			[
+				'component' => 'textarea',
+				'textareaId' => $this->getSettingsName('textarea-required-id'),
+				'textareaFieldLabel' => \__('textarea required label', 'eightshift-forms'),
+				'textareaFieldHelp' => __('help field textarea required', 'eightshift-forms'),
+				'textareaValue' => $this->getOptionValue('textarea-required-id'),
+				'textareaIsRequired' => true,
+			],
+			[
+				'component' => 'select',
+				'selectId' => $this->getSettingsName('select-id'),
+				'selectFieldLabel' => \__('select label', 'eightshift-forms'),
+				'selectFieldHelp' => __('help field select', 'eightshift-forms'),
+				'selectValue' => $this->getOptionValue('select-id'),
+				'selectOptions' => [
+					[
+						'component' => 'select-option',
+						'selectOptionLabel' => 'select-label1',
+						'selectOptionValue' => 'select-id1',
+						'selectOptionIsSelected' => $this->getOptionValue('select-id') === 'select-id1',
+					],
+					[
+						'component' => 'select-option',
+						'selectOptionLabel' => 'select-label2',
+						'selectOptionValue' => 'select-id2',
+						'selectOptionIsSelected' => $this->getOptionValue('select-id') === 'select-id2',
+					],
+				]
+			],
+			[
+				'component' => 'select',
+				'selectId' => $this->getSettingsName('select-required-id'),
+				'selectFieldLabel' => \__('select required label', 'eightshift-forms'),
+				'selectFieldHelp' => __('help field select required', 'eightshift-forms'),
+				'selectValue' => $this->getOptionValue('select-required-id'),
+				'selectIsRequired' => true,
+				'selectOptions' => [
+					[
+						'component' => 'select-option',
+						'selectOptionLabel' => '---',
+						'selectOptionValue' => '',
+						'selectOptionIsSelected' => $this->getOptionValue('select-required-id') === '',
+					],
+					[
+						'component' => 'select-option',
+						'selectOptionLabel' => 'select-label1',
+						'selectOptionValue' => 'select-id1',
+						'selectOptionIsSelected' => $this->getOptionValue('select-required-id') === 'select-id1',
+					],
+					[
+						'component' => 'select-option',
+						'selectOptionLabel' => 'select-label2',
+						'selectOptionValue' => 'select-id2',
+						'selectOptionIsSelected' => $this->getOptionValue('select-required-id') === 'select-id2',
+					],
+				]
+			],
+			[
+				'component' => 'checkboxes',
+				'checkboxesFieldLabel' => \__('checkboxes', 'eightshift-forms'),
```

Comment:
Global prefix on translation function looks odd.

Reply:
fixed in the other PR